barretenberg/cpp/src/barretenberg/stdlib/goblin_verifier/goblin.test.cpp

```diff
@@ -41,7 +41,7 @@ TEST_F(GoblinTests, MultipleCircuits)
     size_t NUM_CIRCUITS = 3;
```

Comment:
Its debatable whether or not this test suite and the two that follow can just be deleted entirely. There's surely a bit of useful stuff going on in some of them but for the most part they're hold overs from the old days of vanilla Goblin IVC.

Reply:
I guess eventually we can delete them, but right now I still find them a tiny bit useful as a unit test for Goblin